app.js

```diff
@@ -21,19 +21,21 @@ config.loadConfig()
 		exit(1)
 	})
 
-// NOTE might have to blacklist things like robots.txt and this from /room names
+// -- NOTE -- might have to blacklist things like robots.txt and this from /room names
 app.use('/favicon.ico', express.static(path.join(__dirname, 'public', 'images', 'favicon.ico')));
 
-// FIXME are these both needed?
+// JSON engine ExpressJS
```

Comment:
We'd need to test removing one of these two:
```js
app.use(express.json())
app.use(express.urlencoded({ extended: false }))
```
or
```js
app.use(bodyParser.json()) // support json encoded bodies
app.use(bodyParser.urlencoded({ extended: true })) // support x-www-form-urlencoded bodies
```

Because I'm not sure if we actually need both of these blocks, or just one. Did you check? If not, we should change it back to the original comment so that eventually it's tested

Reply:
I haven't checked that, maybe I might try.

Reply:
Oh, wait! `bodyParser` is deprecated, I'm removing it,

Reply:
Does removing it break any uploading functionality? What are you replacing it with?

Reply:
I haven't replaced with anything, it's deprecated, but it works fine